docs/src/glossary.md.in

```diff
@@ -196,7 +196,8 @@ See also the [emit-statements section](reference-dsl-output-statements.md#emit-s
 ## empty
 
 Refers to the string with zero characters. For example, in a CSV file with [header line](#header)
-`a,b,c` and data
+`a,b,c` and data `,,` the three fields are empty; with data `1,2,` the first two fields (`a` and `b`)
```

Comment:
I always use the Oxford comma -- let's please keep this consistent.x

Reply:
@Poshi I can assure you it's a long-standing issue in English style, and I'm in the Oxford comma camp. :)

You're right, though, I flagged it here when I shouldn't have -- there aren't three things. Sorry to confuse! :(

Reply:
Hahahahaha! No problem. I'm not in any side, I have no opinion on this stylistic decision.
In spanish the Oxford comma does not exist and I never used/saw it, so I didn't thought about it when doing my modifications. For sure, the best course is to keep the style consistent and following the desires of the creator :-)